frontend/control-center/src/pages/Connectors/index.tsx

```diff
@@ -77,6 +76,11 @@ const Connectors = (props: ConnectedProps<typeof connector>) => {
         ) : (
           <>
             {sourcesInfo.map((infoItem: SourceInfo, index: number) => {
+              console.log(
```

Comment:
Note: `Connectors` is still using `SourcesInfo`, but we will get rid of this mock component soon (#3560) and only use the info from `components.get` and `components.list`